Spanner/src/SpannerClient.php

```diff
@@ -73,6 +73,26 @@
  * $spanner = new SpannerClient();
  * ```
  *
+ * ```
+ * use Google\Cloud\Spanner\SpannerClient;
+ * use Google\Cloud\Spanner\V1\DirectedReadOptions\ReplicaSelection\Type as ReplicaType;
+ *
+ * $spanner = new SpanneClient();
+ * $directedOptions = [
+ *     'directedReadOptions' => [
+ *         'includeReplicas' => [
+ *             'replicaSelections' => [
+ *                 [
+ *                     'location' => 'us-central1',
```

Comment:
Q: if a user chooses a certain replica location, can they see different results in query time or they are guaranteed to be same from the backend?

Reply:
If the CRUD query results may vary, then we should mention that in the constructor docblocks

Reply:
This is only applicable for read operations and the result will be same.